wordpress-mobile/gutenberg-mobile
# Add convert to regular blocks action to Reusable block

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/31012

To test:
Check testing instructions in the [`gutenberg` PR](https://github.com/WordPress/gutenberg/pull/31012).

PR submission checklist:
